wlauto/external/uiauto/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-17
+target=android-18
```

Comment:
I'm not comfortable merging `googlephotos` workload until it's assets are available, as we'll have a workload that users are unable to run. Please move the commit into a separate pull request.